deps/graph-parser/src/logseq/graph_parser/block.cljs

```diff
@@ -666,14 +678,8 @@
 
                 (heading-block? block)
                 (let [block' (construct-block block properties timestamps body encoded-content format pos-meta with-id? options)
-                      block'' (assoc block' :macros (extract-macros-from-ast (cons block body)))
-                      block-uuid (:uuid block'')
-                      fixed-block (if (or (@*block-ids block-uuid)
-                                          (block-exists-in-another-page? db block-uuid page-name))
-                                    (fix-duplicate-id block'')
```

Comment:
moved to https://github.com/logseq/logseq/pull/8823/files#diff-a1e2a20e287e81626ce5ec90828dbc2d0d6c789401d44eeb842b5ffe0185038aR167

Reply:
and https://github.com/logseq/logseq/pull/8823/files#diff-59a4f1336962f1768912740687a6bee88d98d2df45d059e681e73b493e6aa15aR633